src/arm/linux/api.h

```diff
@@ -138,6 +138,7 @@ struct cpuinfo_arm_linux_proc_cpuinfo_cache {
 #define CPUINFO_ARM_LINUX_FEATURE2_RNG UINT32_C(0x00010000)
 #define CPUINFO_ARM_LINUX_FEATURE2_BTI UINT32_C(0x00020000)
 #define CPUINFO_ARM_LINUX_FEATURE2_SME UINT32_C(0x00800000)
+#define CPUINFO_ARM_LINUX_FEATURE2_SME2 UINT64_C(0x0000002000000000)
```

Comment:
Might as well add [others](https://github.com/torvalds/linux/blob/abf2050f51fdca0fd146388f83cddd95a57a008d/arch/arm64/include/uapi/asm/hwcap.h#L99-L104) SME[2] fields?

Reply:
I've added the bits, should I also add them to `cpuinfo_isa`, with the corresponding `cpuinfo_has_arm_*` functions?

Reply:
let me know if you are doing this else I will merge this tomorrow.

Reply:
I'll add them!

Reply:
Done :slightly_smiling_face: